emr-logservice/src/main/java/org/apache/spark/streaming/aliyun/logservice/constants/Constants.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.spark.streaming.aliyun.logservice.constants;
+
+import org.apache.spark.streaming.aliyun.logservice.utils.VersionInfoUtils;
+
+/**
+ * @author jiashuangkai
+ * @version 1.0
+ * @since 2019-09-27 14:56
+ */
+public interface Constants {
+    String LOG_CONNECTOR_USER_AGENT = VersionInfoUtils.getDefaultUserAgent();
```

Comment:
Is there any need to add this new class?

Reply:
Many classes use this constant

Reply:
you can use "VersionInfoUtils.getDefaultUserAgent()" directly

Reply:
ok